### DISCUSSION TITLE
Condicionales en HTML

### DISCUSSION THREAD

**Discussion by ErickBarreraO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#231] [Q&A]
Hola buenas tardes, para la práctica de la pastelería he pensado en agregar algunos condicionales como los que vimos en python, investigué un poco al respecto y lo que encontre es que eso se hace con JavaScript, mi duda es ¿Solo se puede hacer a través de JS o con el puro HTML hay alguna forma?
Gracias.

**Comment by davidtoString** [comment]
Solo a través de JS

**Comment by ABD2k** [comment]
Solo con JavaScript , css tiene un tipo de condicionales pero la mejor opciones JS , HTML solo es un lenguaje para crear un esqueleto

**Comment by JuanSalvador73** [comment]
Solo se puede con JavaScript. HTML es solo un lenguaje de etiquetado. No está hecho para programación, de hecho es un error muy común creer que es programación. 

Saludos

**Comment by carlos-gmx3** [comment]
Puedes jugar con las pseudo clases en css para crear cierta respuesta a acciones como un menú desplegable al presionar un boton o cambiar el color y tamaño de un botón al pasar el mouse sobre el y crear algunas animaciones pero hasta ahí con html y css
JS es lo que le da vida y funcionalidad a un sitio web, es el lenguaje de la web, los mismo navegadores solo aceptan JS y si boen hay experiementos donde se trata de incorporar programación con Python y otros lenguajes, aún no hay soporte para que esto se pueda ver en cualquier dispositivo.
